tests/appsec/integrations/pygoat_tests/Dockerfile.pygoat.2.0.1

```diff
@@ -31,6 +31,9 @@ WORKDIR /app/pygoat
 RUN git checkout v2.0.1
 RUN python3 -m pip install --no-cache-dir pip==22.0.4
 RUN pip install --no-cache-dir -r requirements.txt
+# At some point, pyyaml started failing on CI with CBaseLoader import error, this
+# fixed the problem
+RUN pip --no-cache-dir install --force-reinstall -I pyyaml
```

Comment:
Better pinning to some specific version to avoid surprises when pyyaml gets updated?

Reply:
Good idea

Reply:
Done